File: POOJAVA/src/clasesPOO/Docente.java
package clasesPOO;

public class Docente 
{
    String idDoc;
    int idPers;
    String idCurso;
    public Docente(String idDo)
    {
        establecerDatos(idDo);
    }
    public String ObteneridDoc()
    {
       return idDoc;
    }
   void  establecerDatos(String idD)
    {
        idDoc=idD;
    }
}

File: POOPHP/app/Model/Docente.php
<?php
namespace App\Model;
class Docente
{
    public $idDoc="";
    public $idPers="";
    public $idCurso="";
   public function __construct(String $idDo)
    {
        $this->establecerDatos($idDo);
    }
    public function ObteneridDoc()
    {
        return $this->idDoc;
    }
    public function establecerDatos(String $idD)
    {
        $this->idDoc=$idD;
    }
}
?>